src/utils/components/ListsLists/onConfirm.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @param {number} id
+ * @param {function} callback
+ * @returns {function(): *}
+ */
+const onConfirm = (id, callback) => () => {
+  callback({ id })
+}
+
+export default onConfirm
```

Comment:
looks like overhead

Reply:
I use this in two cases, replaced to shared